cmd/adminapi/main.go

```diff
@@ -87,15 +88,22 @@ func realMain(ctx context.Context) error {
 	}
 	r.Use(httplimiter.Handle)
 
+	// Create the renderer
+	h, err := render.New(ctx, "", config.DevMode)
```

Comment:
DevMode exists on all the configs now. If enabled on an API server, it will print debug messages if JSON marshalling or rendering fails.